scripts/cnv_cromwell_tests/somatic/cnv_somatic_pair_wes_do-gc_workflow_funcotator.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is one test JSON enough?  It doesn't have all the arguments in it that are supported, so I'm wondering if it makes sense to explicitly put them in (or have multiple test cases).

Reply:
This is generally blocked by an inability to easily examine the outputs of a given cromwell run.  This is captured in #6013 

No action.